clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -28,20 +33,72 @@ func createSimpleTestExecutor() TestExecutor {
 
 // ExecuteTest executes test based on provided configuration.
 func (ste *simpleTestExecutor) ExecuteTest(ctx Context, conf *api.Config) error {
+	err := ctx.GetFramework().CreateAutomanagedNamespaces(int(conf.AutomanagedNamespaces))
+	defer cleanupResources(ctx)
```

Comment:
Could you push this line to the top of function?

Reply:
done